app/views/handlers/confirmation_email.py

```diff
@@ -27,5 +34,15 @@ def get_page_title(self):
     @staticmethod
     def handle_post():
         session_store = get_session_store()
-        session_store.session_data.confirmation_email_sent = True
+        session_store.session_data.confirmation_email_count += 1
         session_store.save()
+
+
+def email_limit_reached():
+    session_store = get_session_store()
+
+    return (
+        session_store
+        and session_store.session_data.confirmation_email_count
+        >= current_app.config["CONFIRMATION_EMAIL_LIMIT"]
+    )
```

Comment:
I think we could make this a little neater if we made this a static method within the `ConfirmationEmail` class. That way we wouldn't need to import `email_limit_reached` in `questionnaire.py` - we could call `ConfirmationEmail.is_limit_reached()` instead.

```suggestion

    @staticmethod
    def is_limit_reached():
        session_store = get_session_store()

        return (
            session_store
            and session_store.session_data.confirmation_email_count
            >= current_app.config["CONFIRMATION_EMAIL_LIMIT"]
        )
```